packages/fxa-auth-server/config/index.ts

```diff
@@ -2213,6 +2213,18 @@ const convictConf = convict({
         env: 'RECOVERY_PHONE__SMS__MAX_RETRIES',
         format: Number,
       },
+      simPumpingRiskThreshold: {
```

Comment:
Should this be `smsPumpingRiskThreshold`? + everywhere else it is used
```suggestion
      smsPumpingRiskThreshold: {
```